scripts/release.sh

```diff
@@ -64,10 +64,12 @@ _do_release_commit() {
 
   if ! test "$ARG1" = '--use-current-commit' ; then
     echo "Building changelog since ${__LAST_TAG}..."
-    __CHANGELOG="$(./scripts/git-log-pretty-since.sh "$__LAST_TAG" 'vim-patch:[^[:space:]]')"
+    echo "${__RELEASE_MSG}" >_release.bk
+    ./scripts/git-log-pretty-since.sh "$__LAST_TAG" 'vim-patch:[^[:space:]]' >>_release.bk
 
     git add CMakeLists.txt
-    git commit --edit -m "${__RELEASE_MSG} ${__CHANGELOG}"
+    git commit --edit -F _release.bk
```

Comment:
This would avoid needing to generate a temp file and ensure it gets cleaned up.

```suggestion
    (echo "${__RELEASE_MSG}"; ./scripts/git-log-pretty-since.sh "$__LAST_TAG" 'vim-patch:[^[:space:]]') | git commit --edit -F -
```

Reply:
Works on my machine.